Solutions/CTM360/Analytic Rules/HeaderXContentTypeOptionsMissing.yaml

```diff
@@ -0,0 +1,42 @@
+id: e6fe772a-c669-4b42-b25a-e01ab8ab02f7
+name: 'Header: X-Content-Type-Options Missing'
+severity: Medium

```

Comment:
Hello @edx-sayed-salman description is missing from this detection due to which its failing for solution packaging

Reply:
Hi , @v-prasadboke , we add the description in the rule.